### ISSUE TITLE
new Docker images for microservices are not pulled during upgrade

### ISSUE THREAD

**Issue by PetrHeinz** [shopsys/shopsys#464]
### What is happening

In the project-base, we use pre-built Docker images for microservices in their `latest` version on `master` branch.
If we change the microservices, new Docker image is built and pushed to the Docker Hub, overwriting the current `lastest` tag.

But these new images are then never pulled during execution of `docker-compose up`.
Running `docker-compose pull` is needed to update the images, but this is never mentioned in our docs.

*This will not be an issue with the released versions (as they will have their own tags), only with building the current `master` branch.*

### Expected result

When working with project-base on `master`, updating all microservice Docker images should be recommended for Unix platforms (maybe in the troubleshooting section, in the UPGRADE.md, or as a note in the installation process):
```bash
docker-compose pull $(docker-compose config --services | grep microservice)
```
This command will pull new images for all microservices (while not updating other used images, such as postgres).

On Windows without some linux-like environment (eg. cygwin), I would recommend running just `docker-compose pull` to update all images instead.

**Comment by Miroslav-Stopka** [comment]
Hi @PetrHeinz,

I don't understand, why this scenario should be mentioned in the docs. The installation guide describes the installation for stability = beta. So if there is someone, who does not follow the installation instructions, he should expect consequences. Or are we starting to write documentation even for the scenarios where our proper instructions are not followed?

Can you explain it to me, please?

**Comment by PetrHeinz** [comment]
@Miroslav-Stopka You are probably right about that. I created this issue in the context of our failing CI (which was fixed with `docker-compose pull $(docker-compose config --services | grep microservice)`), and around that time we recommended installing `master` branch due to some unresolved issues. That lead me to consider having a note about this in the docs.

We could probably close this issue. Thanks for noticing :+1:
